forte/processors/base/data_augment_processor.py

```diff
@@ -60,8 +60,11 @@ def replace(self, replacement_op: TextReplacementOp, input: Entry):
         getting the augmented text, it will register the input & output
         for later batch process of building the new data pack.
         """
-        replaced_text: str = replacement_op.replace(input)
-        self.replaced_spans.append((input, replaced_text))
+        replaced_text: str
+        is_replace: bool
+        is_replace, replaced_text = replacement_op.replace(input)
+        if is_replace:
+            self.replaced_spans.append((input, replaced_text))
```

Comment:
I am wondering is it sufficient to record the length of the input and replaced_text?

Reply:
For the input entry, we need the begin and end span. For the replaced_text, we need the text string to build the new datapack. BTW, the data_augment_processor.py here is just a placeholder to pass the Travis test. The real file is in the auto_align_replace PR.